compiler/nimconf.nim

```diff
@@ -254,6 +254,10 @@ proc loadConfigs*(cfg: RelativeFile; cache: IdentCache; conf: ConfigRef; idgen:
     if isMain and optWasNimscript in conf.globalOptions:
       if conf.projectIsStdin: s = stdin.llStreamOpen
       elif conf.projectIsCmd: s = llStreamOpen(conf.cmdInput)
+    if conf.projectFull.string != "" and not fileExists(conf.projectFull):
+      if conf.cmd == cmdNimscript or conf.cmd == cmdUnknown:
+        echo "target file is not existed"
```

Comment:
* not proper english (`doesn't exist` is better)
* should show relevant runtime context (`conf.projectFull`)
* don't use echo (stdout) for `errors`, use `localError`
* this PR makes `nim --eval:'echo 123'` worse by showing `target file is not existed`
* the bug isn't properly fixed:

```
nim foo $timn_D/tests/nim/all/t12217.nonexistant
target file is not existed
Error: invalid command: foo
```

instead should show:
```
nim foo /pathto/t12217.nonexistant
Error: nimscript file '/pathto/t12217.nonexistant' not found
```

Reply:
thaks for reviewing.
I fixed codes.

Reply:
I'm sorry, I misunderstood.
Please ignore it.